### PR TITLE
feat(datagrid) : pagination doesn't render if dataset is less than defaultPageSize 

### PR DESCRIPTION
**Pull request by karthik-sista**

### CODE REVIEW

**Comment by karthik-sista** [comment]
retest this please .

File: libs/datagrid/src/DataGrid.test.js [hunk 1]
```diff
@@ -39,4 +39,8 @@ describe('DataGrid', () => {
     const wrapper = shallow(<DataGrid data={[]} />)
     expect(wrapper.at(0).hasClass('-highlight')).toBe(true)
   })
+  it('doesnot render pagination if the data set is less that defaultPageSie', () => {
```
Comment (mlipardo) [inline libs/datagrid/src/DataGrid.test.js]: nit(misspelled): `does not`

**Comment by karthik-sista** [comment]
Capture
Tested in IE . works as expected. 👍

File: libs/datagrid/src/DataGrid.jsx [hunk 2]
```diff
@@ -6,6 +6,17 @@ import { config } from './DataGridConfig'
...
+  }
+  return false
+}
+
 export default ({ className, ...props }) => (
-  <ReactTable className={cn(className, '-highlight')} {...props} />
+  <ReactTable
+    className={cn(className, '-highlight')}
+    {...props}
+    showPagination={displayPagination(props.data)}
```
Comment (eckdanny-osi) [inline libs/datagrid/src/DataGrid.jsx]: `config()` _sets_ the defaults every time (side effect). We should not call this on renders.

**Comment by eckdanny-osi** [comment]
close in favor of #461?

@karthik-sista ^

**Comment by eckdanny-osi** [comment]
closed in favor of #461

**Pull request closed without merging**